capnp/src/message.rs

```diff
@@ -526,12 +526,12 @@ where
     message: Builder<A>,
 }
 
-impl<T> TypedBuilder<T, HeapAllocator>
+impl<T> Default for TypedBuilder<T, HeapAllocator>
 where
     T: Owned,
 {
-    pub fn new_default() -> Self {
```

Comment:
Abruptly deleting this method would cause unnecessary pain. It would be better to keep it around and deprecate it with a warning like "use the default() method instead (from the Default trait)".